dnf/cli/commands/install.py

```diff
@@ -45,7 +45,7 @@ class InstallCommand(commands.Command):
 
     @staticmethod
     def set_argparser(parser):
-        parser.add_argument('package', nargs='+', metavar=_('PACKAGE'),
+        parser.add_argument('package', nargs='*', metavar=_('PACKAGE'),
```

Comment:
Well, they will be supported, but user will have to type `*` by himself, so the user cannot blame us, that we broke his computer/transaction takes ages and so on.